File: hrm_common_model/src/main/java/com/study/domain/company/Company.java
package com.study.domain.company;

import com.fasterxml.jackson.annotation.JsonFormat;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.io.Serializable;
import java.util.Date;

/**
 * 实体类代码
 */
@Data
@NoArgsConstructor
@Entity
@Table(name = "co_company")
public class Company implements Serializable {
    private static final long serialVersionUID = 594829320797158219L;
    @ApiModelProperty(value = "主键id", required = true,hidden = true)
    @Id
    private String id;
    @ApiModelProperty(value = "公司名称")
    private String name;                    // 公司名称
    @ApiModelProperty(value = "企业登录账号ID", required = true)
    private String managerId;               // 企业登录账号ID
    @ApiModelProperty(value = "当前版本")
    private String version;                 // 当前版本
    @ApiModelProperty(value = "续期时间")
    private Date renewalDate;               // 续期时间
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    @ApiModelProperty(value = "到期时间")
    private Date expirationDate;            // 到期时间
    @ApiModelProperty(value = "公司地区")
    private String companyArea;             // 公司地区
    @ApiModelProperty(value = "公司地址")
    private String companyAddress;          // 公司地址
    @ApiModelProperty(value = "营业执照-图片id")
    private String businessLicenseId;       // 营业执照-图片id
    @ApiModelProperty(value = "法人代表")
    private String legalRepresentative;     // 法人代表
    @ApiModelProperty(value = "公司电话")
    private String companyPhone;            // 公司电话
    @ApiModelProperty(value = "邮箱")
    private String mailbox;                 // 邮箱
    @ApiModelProperty(value = "公司规模")
    private String companySize;             // 公司规模
    @ApiModelProperty(value = "所属行业")
    private String industry;                // 所属行业
    @ApiModelProperty(value = "备注")
    private String remarks;                 // 备注
    @ApiModelProperty(value = "审核状态")
    private String auditState;              // 审核状态
    @ApiModelProperty(value = "状态")
    private Integer state;                  // 状态
    @ApiModelProperty(value = "余额")
    private Double balance;                 // 余额
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
    @ApiModelProperty(value = "创建时间")
    private Date createTime;                // 创建时间
}

File: project-saas-hrm-vue-master/src/mock/company.js
import Mock from 'mockjs'
import { param2Obj } from '@/utils'

const List = []
const count = 100

for (let i = 0; i < 10; i++) {
    let data = {
        id: '1' + i,
        name: '企业' + i,
        managerId: 'string',
        version: '试用版v1.0',
        renewalDate: '2018-01-01',
        expirationDate: '2019-01-01',
        companyArea: '山东济南历下区',
        companyAddress: '力高国际',
        businessLicenseId: 'string',
        legalRepresentative: 'mr.yangtk',
        companyPhone: '18854000080',
        mailbox: 'yangtongkuan@qq.com',
        companySize: '15人以下',
        industry: 'string',
        remarks: 'string',
        auditState: '1',
        state: '1',
        balance: '0.0',
        createTime: '2020-12-12'
    }
  List.push(data)
}

export default {
  list: () => {
    return {
        code: 10000,
        success: true,
        message: '查询成功',
        data: List
    }
  },
  sassDetail: () => {
    return {
      code: 10000,
      success: true,
      message: '查询成功',
      data: {
        id: '10001',
        name: '测试企业',
        managerId: 'string',
        version: '试用版v1.0',
        renewalDate: '2018-01-01',
        expirationDate: '2019-01-01',
        companyArea: 'string',
        companyAddress: 'string',
        businessLicenseId: 'string',
        legalRepresentative: 'string',
        companyPhone: '13800138000',
        mailbox: 'string',
        companySize: 'string',
        industry: 'string',
        remarks: 'string',
        auditState: 'string',
        state: '1',
        balance: 'string',
        createTime: 'string'
      }
    }
  }
}
